### PR TITLE
[API-1528] rate limiting documentation

### PR DESCRIPTION
**Pull request by zmomen-cobalt**
[Ticket [API-1528]](https://zombie.atlassian.net/browse/API-1528)

[API-1528]: https://zombie.atlassian.net/browse/API-1528?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by grayson-cobalt** [comment]
vale is so annoying. maybe we get rid of it since none of us resolve the warnings anyways

File: versions/v2/content/rate-limits.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+weight: 15
+title: Rate Limits
+---
+
+# Rate Limits
+
+Cobalt API provides rate limiting on API requests to manage request volume and provide a reliable and available API.
```
Comment (grayson-cobalt) [inline versions/v2/content/rate-limits.md:8]: `Cobalt API provides rate limiting`. This is pitching it like it's a feature. More succinct:
`Cobalt API rate limits API requests ...`

File: versions/v2/content/rate-limits.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Rate Limits
+
+| Request Type    | HTTP Verb                     | Limit | Window     | HTTP Response Headers                                   |
+|-----------------|-------------------------------|-------|------------|---------------------------------------------------------|
+| Get Requests    | `GET`                         | 100   | 60 seconds | `X-Rate-Limit-Get-Limit` and  `X-Rate-Limit-Remaining`  |
+| Mutate Requests | `POST` `PATCH` `PUT` `DELETE` | 20    | 60 seconds | `X-Rate-Limit-Mutate-Limit`and `X-Rate-Limit-Remaining` |
+
+## HTTP Response Headers
+
+- `X-Rate-Limit-Get-Limit`: indicates the maximum rate allowed for `GET` requests.
```
Comment (grayson-cobalt) [inline versions/v2/content/rate-limits.md:24]: Maybe:
`indicates the maximum number of GET requests allowed per minute.`
?

File: versions/v2/content/rate-limits.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Example: `X-Rate-Limit-Get-Limit: 100`
+- `X-Rate-Limit-Mutate-Limit`: indicates the maximum rate allowed for Mutate requests.
+Example: `X-Rate-Limit-Mutate-Limit: 20`
+- `X-Rate-Limit-Remaining`: indicates the number of requests left in the window of time.
+Example: `X-Rate-Limit-Remaining: 93`
+
+## When Rate Limit is Exceeded
+
+When a user exceeds the rate limit, the API will return HTTP `509 BANDWIDTH_LIMIT_EXCEEDED` response code and message
+along with the headers above. The user will have to wait no more than the window of time above in order for the
```
Comment (grayson-cobalt) [inline versions/v2/content/rate-limits.md:34]: `The user will have to wait no more than the window of time above`
Can we say this more succinctly?

File: versions/v2/content/rate-limits.md (hunk 1 above)
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:8]:
```suggestion
The Cobalt API sets rate limits on API requests to manage request volume and maintain system stability.
```

File: versions/v2/content/rate-limits.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+# Rate Limits
+
+Cobalt API provides rate limiting on API requests to manage request volume and provide a reliable and available API.
+Here are some details on the rate limit policy that Cobalt API implements:
+
+- Rate limiting follows a **Fixed-Window policy** where there is a maximum amount of requests allowed in a specific
+window of time.
+- Rate limiting is applied on a **per-user basis**, regardless of how many API tokens the user owns.
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:13]:
```suggestion
- Rate limiting applies on a **per-user basis**, regardless of how many API tokens the user has.
```

File: versions/v2/content/rate-limits.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Rate Limits
+
+| Request Type    | HTTP Verb                     | Limit | Window     | HTTP Response Headers                                   |
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:17]:
```suggestion
| Request Type    | HTTP Verbs                     | Limit | Window     | HTTP Response Headers                                   |
```

File: versions/v2/content/rate-limits.md (hunk 2 above)
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:24]:
```suggestion
- `X-Rate-Limit-Get-Limit`: Indicates the maximum number of `GET` requests allowed per minute.
```

File: versions/v2/content/rate-limits.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+|-----------------|-------------------------------|-------|------------|---------------------------------------------------------|
+| Get Requests    | `GET`                         | 100   | 60 seconds | `X-Rate-Limit-Get-Limit` and  `X-Rate-Limit-Remaining`  |
+| Mutate Requests | `POST` `PATCH` `PUT` `DELETE` | 20    | 60 seconds | `X-Rate-Limit-Mutate-Limit`and `X-Rate-Limit-Remaining` |
+
+## HTTP Response Headers
+
+- `X-Rate-Limit-Get-Limit`: indicates the maximum rate allowed for `GET` requests.
+Example: `X-Rate-Limit-Get-Limit: 100`
+- `X-Rate-Limit-Mutate-Limit`: indicates the maximum rate allowed for Mutate requests.
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:26]:
```suggestion
- `X-Rate-Limit-Mutate-Limit`: Indicates the maximum number of Mutate requests allowed per minute.
```

File: versions/v2/content/rate-limits.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+Example: `X-Rate-Limit-Mutate-Limit: 20`
+- `X-Rate-Limit-Remaining`: indicates the number of requests left in the window of time.
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:28]:
```suggestion
- `X-Rate-Limit-Remaining`: Indicates the number of allowed requests remaining in the time window.
```

**Comment by zmomen-cobalt** [comment]
> vale is so annoying. maybe we get rid of it since none of us resolve the warnings anyways

hmmm .. @ana-dashuk-cobalt

**Comment by ana-dashuk-cobalt** [comment]
> > vale is so annoying. maybe we get rid of it since none of us resolve the warnings anyways
> 
> hmmm .. @ana-dashuk-cobalt

@zmomen-cobalt I actually run vale locally. We have many pages in the product docs repo, so it would definitely annoy me, I understand your concern.
You can use vale in your IDE. I'll send you a link to our internal Confluence space for instructions.

File: versions/v2/content/rate-limits.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+---
+weight: 15
+title: Rate Limits
+---
+
+# Rate Limits
+
+The Cobalt API sets rate limits API requests to manage request volume and maintain system availability.
+Here are some details on the rate limit policy that Cobalt API implements:
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:9]:
```suggestion
```

**Review by ana-dashuk-cobalt — DISMISSED** [review]

File: versions/v2/content/rate-limits.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+---
+weight: 15
+title: Rate Limits
+---
+
+# Rate Limits
+
+The Cobalt API sets rate limits API requests to manage request volume and maintain system availability.
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:8]:
```suggestion
The Cobalt API sets rate limits on API requests to manage request volume and maintain system availability.
```

Comment (ana-dashuk-cobalt) [reply versions/v2/content/rate-limits.md:8]: This was a typo in my previous suggestion.

File: versions/v2/content/rate-limits.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+| Get requests    | `GET`                            | 100   | 60 seconds | `X-Rate-Limit-Get-Limit` and  `X-Rate-Limit-Remaining`   |
+| Mutate requests | `POST`, `PATCH`, `PUT`, `DELETE` | 20    | 60 seconds | `X-Rate-Limit-Mutate-Limit` and `X-Rate-Limit-Remaining` |
+
+## HTTP Response Headers
+
+- `X-Rate-Limit-Get-Limit`: Indicates the maximum number of `GET` requests allowed per time window.
+Example: `X-Rate-Limit-Get-Limit: 100`
+- `X-Rate-Limit-Mutate-Limit`: Indicates the maximum number of Mutate requests allowed per time window.
+Example: `X-Rate-Limit-Mutate-Limit: 20`
+- `X-Rate-Limit-Remaining`: Indicates the number of requests remaining in the time window
```
Comment (ana-dashuk-cobalt) [inline versions/v2/content/rate-limits.md:27]:
```suggestion
- `X-Rate-Limit-Remaining`: Indicates the number of requests remaining in the time window.
```

**Review by ana-dashuk-cobalt — APPROVED** [review]

**Pull request merged**